configuration/pih/petl/jobs/sql/schemas/mch_kellogg_data.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE mch_kellogg_data
```

Comment:
I really don't like calling this mch_kellogg_data.  Can we create a table with useful data that is not tied to a particular funder?  Can we just call this mch_j9_data?

Reply:
agreed.  will work with the reporting team to come up with something better.